### PR TITLE
DataUnion types

### PR DESCRIPTION
**Pull request by teogeb**
Types for `getAdminAddress`, `setAdminFee`, `executeWithdraw`. After these modifications, and PR https://github.com/streamr-dev/streamr-client-javascript/pull/207 all public methods of `DataUnion` are annotated with valid types.

Remove `deployTxReceipt` field which was not used

### CODE REVIEW

File: src/dataunion/DataUnion.ts [hunk 1]
```diff
@@ -296,7 +296,7 @@ export class DataUnion {
         return +adminFeeBN.toString() / 1e18
     }
 
-    async getAdminAddress(): Promise<Todo> {
+    async getAdminAddress(): Promise<string> {
```
Comment (jtakalai) [inline src/dataunion/DataUnion.ts:299]: did we have the EthereumAddress type alias now?

File: src/dataunion/DataUnion.ts [hunk 2]
```diff
@@ -446,14 +446,14 @@ export class DataUnion {
     /**
      * Admin: set admin fee (between 0.0 and 1.0) for the data union
      */
-    async setAdminFee(newFeeFraction: number): Promise<Todo> {
+    async setAdminFee(newFeeFraction: number): Promise<void> {
```
Comment (jtakalai) [inline src/dataunion/DataUnion.ts:449]: why not return the receipt?

**Review by jtakalai — APPROVED** [review]

File: src/dataunion/DataUnion.ts (hunk 1 above)
Comment (teogeb) [reply src/dataunion/DataUnion.ts]: Fixed

File: src/dataunion/DataUnion.ts (hunk 2 above)
Comment (teogeb) [reply src/dataunion/DataUnion.ts]: Fixed

**Pull request merged**